lib/top_secret/text.rb

```diff
@@ -56,7 +57,7 @@ def self.filter(input, custom_filters: [], **filters)
     #   ip_filter = TopSecret::Filters::Regex.new(label: "IP", regex: /\d+\.\d+\.\d+\.\d+/)
     #   result = TopSecret::Text.filter_all(messages, custom_filters: [ip_filter])
     def self.filter_all(messages, custom_filters: [], **filters)
-      shared_model = Mitie::NER.new(TopSecret.model_path)
+      shared_model = TopSecret.model_path ? Mitie::NER.new(TopSecret.model_path) : nil
```

Comment:
Now that we have a default model, this line might not be necessary at all. We initialize `@model` with the given model (null or MITIE) and then we just use that variable going forward.